modules/password/password.go

```diff
@@ -66,7 +66,7 @@ func Generate(n int) (string, error) {
 			}
 			buffer[j] = validChars[rnd.Int64()]
 		}
-		if IsComplexEnough(string(buffer)) {
+		if IsComplexEnough(string(buffer)) && (string(buffer[0]) != " " || string(buffer[n-1]) != " ") {
```

Comment:
```suggestion
if IsComplexEnough(string(buffer)) && string(buffer[0]) != " " && string(buffer[n-1]) != " " {
```
The space character must be avoided from both the start _and_ the end of the string.